docs/controls/BubbleMultipleDataset.md

```diff
@@ -0,0 +1,274 @@
+    }
+    color: Carbon.helpers.COLORS.BLUE,
+    values: [
+        {
+            x: "2016-02-03T12:00:00Z",
+            y: 4,
+            weight: 15
+        },
+        {
+            x: "2016-05-01T12:00:00Z",
+            y: 15,
+            weight: 20
+        },
+        {
+            x: "2016-10-01T12:00:00Z",
+            y: 10,
+            weight: 40
+        }
+    ]
+};
+var bubbleWeight = Carbon.api.graph(root);
+bubbleWeight.loadContent(Carbon.api.bubbleMultipleDataset(dataWeightBased));
+```
+
+#### Colored Bubbles
+
+For colored Bubbles, provide `color`:
+
+```javascript
+var colorBasedData = {
+    key: "uid_2",
+    label: {
+        display: "Data Label 2"
+    },
+    color: BUBBLE.PALETTE.ORANGE,
+    values: [
+        {
+            x: "2016-03-01T12:00:00Z",
+            y: 14
+        },
+        {
+            x: "2016-04-10T12:00:00Z",
+            y: 1
+        },
+        {
+            x: "2016-11-01T12:00:00Z",
+            y: 18
+        }
+    ]
+};
+bubbleColorBased = Carbon.api.graph(root);
+bubbleColorBased.loadContent(Carbon.api.bubbleMultipleDataset(colorBasedData));
+```
+
+#### Custom Size Based Bubble
+
+For a custom size bubble, provide `maxRadius` in weight as given below between (min and max) and maxRadius maxRadius is given precedence.
+
+```javascript
+var customSizeBasedData = {
+    key: "uid_2",
+    label: {
+        display: "Data Label 2"
+    },
+    weight: {
+        maxRadius: 10
+    }
+    values: [
+        {
+            x: "2016-03-01T12:00:00Z",
+            y: 14,
+        },
+        {
+            x: "2016-04-10T12:00:00Z",
+            y: 1,
+        },
+        {
+            x: "2016-11-01T12:00:00Z",
+            y: 18,
+        }
+    ]
+};
+bubbleGraph = Carbon.api.graph(root);
+bubbleGraph.loadContent(Carbon.api.bubbleMultipleDataset(customSizeBasedData));
+```
+
+## JSON Properties
+
+### Root
+
+Refer [Graph](../core/Graph.md) `Root` for more details.
+
+### Data
+
+#### Required
+
+| Property Name | Expected | Description                             |
+| ------------- | -------- | --------------------------------------- |
+| key           | string   | Unique id which represents the data-set |
+| values        | Array    | [Values](#values)                       |
+
+#### Optional
+
+| Property Name | Expected | Default      | Description                                                                               |
+| ------------- | -------- | ------------ | ----------------------------------------------------------------------------------------- |
+| color         | string   | COLORS.BLACK | Color for the bubbles                                                                     |
+| onClick       | Function | undefined    | Any action that can be performed when clicking on the data point                          |
+| hue           | object   | {}           | Provide lowerShade and upperShadefor color based bubble [Hue](#Hue)                       |
```

Comment:
Are we still using hue property?

Reply:
Removed:
https://github.com/cerner/carbon-graphs/pull/308/commits/1a920bb9dc6643049ba404be1a3bcc8cd672e8ac